test/stmt/if_unexpected_else.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %target-typecheck-verify-swift
```

Comment:
Sounds good! For future reference, I tried using `target-parse-verify-swift`, but as far as I could tell there wasn't a lit substitution with that name. `swift -frontend -parse -verify`, maybe? But then I think that failed the test because it exited with a non-zero status. I'm not sure `not swift -frontend -parse -verify` would work either, because then presumably the test would pass all the time... 🤔